### ISSUE TITLE
MCP server doesn't start on reboot

### ISSUE THREAD

**Issue by hakamine** [artefactual-labs/ansible-role-archivematica-src#52]
upstart script broke ?

**Comment by hakamine** [comment]
It seems that replacing  `/etc/init/archivematica-mcp-server.conf` with this could work:
```
description     "Archivematica MCP Server"
author          "Artefactual Systems"

start on (started gearman-job-server)
stop on runlevel [016]

env CONF=/etc/archivematica/MCPServer
env LOCATION=/usr/lib/archivematica/MCPServer/archivematicaMCP.py

setuid archivematica
setgid archivematica

pre-start script

    # Check that $CONF directory exists
    [ -d $CONF ]

    # Wait for Gearman to start - timeout eventually
    for i in $(seq 1 10)
    do
        gearadmin --getpid && break
        sleep 3
    done
    gearadmin --getpid || exit 1

    # Wait for mysql to start - timeout eventually
    for i in $(seq 1 10)
    do
        mysqladmin ping && break
        sleep 3
    done
    mysqladmin ping || exit 1

    exit 0

end script

script

    # Run
    $LOCATION

end script
```

**Comment by sevein** [comment]
I'd rather see something like this included in AM 1.5 to be honest: https://bitbucket.org/sevein/arkinematics/commits/1d76b314d43eddb0ba30a54447681ac3b1999aab?at=qa/1.x - but maybe it's too late for that.

**Comment by hakamine** [comment]
For the current 1.5 and earlier: fix in https://github.com/artefactual-labs/ansible-role-archivematica-src/pull/54
For 1.6+: this will probably be handled in the archivematica code as mentioned by @sevein
